pkg/odo/cli/component/common_link.go

```diff
@@ -228,6 +246,24 @@ func (o *commonLinkOptions) validate(wait bool) (err error) {
 
 func (o *commonLinkOptions) run() (err error) {
 	if experimental.IsExperimentalModeEnabled() {
+		if o.operationName == "unlink" {
```

Comment:
are we actually using anything from the `common_link` code? seems quite disjoint to me?

Reply:
Discussed this over the call so not touching this part.